components/logins/src/login.rs

```diff
@@ -2,6 +2,253 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+//  N.B. if you're making a documentation change here, you might also want to make it in:
+//
+//    * The API docs in ../ios/Logins/LoginRecord.swift
+//    * The API docs in ../android/src/main/java/mozilla/appservices/logins/ServerPassword.kt
+//    * The android-components docs at
+//      https://github.com/mozilla-mobile/android-components/tree/master/components/service/sync-logins
+//
+//  We'll figure out a more scalable approach to maintaining all those docs at some point...
```

Comment:
I realize I have not followed my own advice here, but I can once we're happy with the descriptions, if it seems valuable. Or, we could consider moving this out into a markdown document and just putting pointers at it in the various API docs.